cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,98 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import faker from 'faker';
+import {
+  updateSuccessMessage,
+  confirmationMessage
+} from '../plugins/alertMessages';
+
+const signInPage = new SignInPageObject();
+const homePage = new HomePageObject();
+const settingsPage = new SettingsPageObject();
 
 describe('Settings page', () => {
-  before(() => {
+  let user;
 
+  before(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+      cy.registerUser(user.email, user.username, user.password);
+    });
   });
 
   beforeEach(() => {
+    signInPage.visit();
+  });
 
+  afterEach(() => {
+    settingsPage.logOutSettings();
   });
 
   it('should provide an ability to update username', () => {
+    signInPage.login(user.email, user.password);
+    homePage.clickSettings();
+
+    const editedUsername = faker.lorem.words(1);
+    settingsPage.usernameInput.clear().type(editedUsername);
+    settingsPage.submitSettings();
+
+    cy.contains(updateSuccessMessage);
```

Comment:
You are not verifying if 'updateSuccessMessage' and 'confirmationMessage' are present on the page with an assertion. This could lead to false positives.

Reply:
Done